### ISSUE TITLE
subtitle loaded at a wrong timestamp on Disney Plus

### ISSUE THREAD

**Issue by fredflys** [killergerbah/asbplayer#576]
**Before submitting an issue check out the common issues section of the README:**
https://github.com/killergerbah/asbplayer?tab=readme-ov-file#common-issues

**What is the issue and how what do you do to make it happen**
I was streaming a video on Disney+ and there is always a timing discrepancy in the loaded subtitles. 
![disney](https://github.com/user-attachments/assets/24a45475-54c0-4811-9531-526b5af7afd5)
For example, the episode loaded was playing at 3:10 but the subtitle loaded was at 0:44, as shown in the screenshot above. 
I checked other bug reports. Some mentioned ads could be the cause as they are considered extra videos. However, I don't see any similar issues when watching on Netflix. 

**What browser?**
Chrome, Edge

**Is it happening with local video or streaming video?**
streaming video

**If the issue is happening with streaming video, what version of the extension are you using?**
1.6.1

**If the issue is happening with streaming video, on what website(s) is the issue happening?**
www.disneyplus.com

**Comment by killergerbah** [comment]
@fredflys Yes this is a known issue and I have seen it happen when the Disney+ logo shows at the beginning of the video. Usually refreshing will work around the problem.

**Comment by fredflys** [comment]
Thank you for getting back to me. It seems the issue persists on my end even after refreshing. For now, I guess I will just try to stream on Netflix a little more.😂 Thank you for the great product!

**Comment by ioridev** [comment]
Hmmm, I tried to refresh mine but the timestamps are still out of sync.

**Comment by ioridev** [comment]
https://chromewebstore.google.com/detail/disney-plus-%E5%AD%97%E5%B9%95%E7%BF%BB%E8%A8%B3-%E3%83%87%E3%83%A5%E3%82%A2%E3%83%AB%E5%AD%97%E5%B9%95/ojhgmkdbdcgmgcioandnlaabnhofbnel

This extension did not misalign the timestamps. What is the difference?

**Comment by killergerbah** [comment]
`video.currentTime` is completely unreliable on Disney Plus probably due to how the video player streams the video. It's unclear how to obtain an accurate timestamp / seek accurately with the `video` APIs available to us. Current thought is to find a way to obtain a reference to Disney Plus's video player, which appears to be "Hive Player."

**Comment by killergerbah** [comment]
Fixed by https://github.com/asbplayer/asbplayer/pull/956
